ONSdigital/sdx-transform-cs
# Enable transformation for e-commerce

## What? and Why?
https://trello.com/c/LJXexVU2/2591-e-commerce-sdx-transformation-for-openroad

Added transformation for e-commerce survey.  Also created framework to handle additional surveys destined for CORD.

## How to test:
Check out the branch of the same name in the following repos:
sdx-validate
sdx-console
sdx-downstream
sdx-transform-cs

Submit example submission in console (187.0001).  Ensure the the files look sensible (PCK, IDBR receipt, images).  The PCK format is vastly different from previous ones so that needs to be verified.  Every qcode should be present regardless of the answer (unanswered questions are '0') so this needs to be tested.
Additionally, the image needs to be reviewed too.  The images generated by the transformer have historically been not excellent (to put it lightly) but as long as they conveyed the information, it's been fine.

## Checklist
